version.properties

```diff
@@ -1 +1 @@
-VERSION_TAG=1.1.2
\ No newline at end of file
+VERSION_TAG=1.1.3
```

Comment:
We haven't been very disciplined about Semantic Versioning but I'd make this 1.2.0 since your adding new functionality.